sdk/src/openssl/temp_signer.rs

```diff
@@ -62,7 +62,7 @@ use crate::{
 /// # Panics
 ///
 /// Can panic if unable to invoke OpenSSL executable properly.
-pub fn get_signer<P: AsRef<Path>>(path: P) -> (RsaSigner, PathBuf) {
+pub fn get_temp_signer<P: AsRef<Path>>(path: P) -> (RsaSigner, PathBuf) {
```

Comment:
this is the internal one, in temp_signer. I renamed this one because if I promote them both to the top level, I think the one that takes existing certs should be called get_signer and this one that makes temporary certs should be the temp one.